packages/server/test/migration/migrations-diff.test.ts

```diff
@@ -0,0 +1,70 @@
+import fs from 'fs'
+import path from 'path'
+
+import { compareDatabases } from './utils/diff'
+import { decrement } from './utils/semver-decrement'
+import { startMessagingServer } from './utils/server'
+
+const TIMEOUT = 30000
+
+describe('Migrations diff tests', () => {
+  const migrationFolder = path.join(__dirname, '../../src/migrations')
+  const migrationFiles = fs.readdirSync(migrationFolder)
+  const migrations = migrationFiles
+    .filter((f) => !isNaN(Number(f.charAt(0)))) // Only keep migration files
+    .map((f) => {
+      const version = f.split('-')[0]
+      return [f, version, decrement(version)]
+    })
```

Comment:
Don't need to do this. The migrations are listed in the packages/server/serc/migrations/index.ts file. You can just import the array from there. The file names of migrations are not significative (by design). They only have the name of the migration and the version for organizational purposes. There is a `meta` property of the base migration class that you can use to get this info.

Reply:
To get instances you can do the same thing the MigrationService does with `const all = this.migs.map((x) => new x())`

Reply:
Right, I'll adapt this.